docs/docs/non-cloud-deployment.md

```diff
@@ -0,0 +1,114 @@
+
+# Non-cloud deployment
+
+Conclave recommends users to [deploy applications in Azure VMs](machine-setup.md) that support the latest 
+attestation protocols. However, you can also deploy your enclaves on-premise.
+
+If your hardware does not support DCAP attestation, you need to get access to the
+[Intel Attestation Service (IAS)](ias.md).
+
+To configure the host:
+
+1. Install the SGX kernel driver (not required for Linux kernel versions later than 5.11).
```

Comment:
I am not sure about this doc. I see that we are talking about IAS and kernel driver: I do not think we should keep mentioning them. Let me know what you think, we might want to refactor this in a separate PR, where we remove references to out of kernel driver and EPID

Reply:
I will create a new ticket for removing EPID and out-of-kernel references. Might need to add new steps for using DCAP in some of the docs. Need to research that.

Reply:
Feel free to contact me for any help. Let's leave EPID here for the moment.